api/src/main/java/com/epam/pipeline/manager/cluster/KubernetesManager.java

```diff
@@ -75,6 +75,8 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(KubernetesManager.class);
     private static final String DEFAULT_SVC_SCHEME = "http";
     private static final int NODE_PULL_TIMEOUT = 200;
+    private static final String NEW_LINE = "\n";
+    private static final String TRUNCATED_SIGN = "< ... Log was truncated, it is too big ...>";
```

Comment:
Please, change the message text to `Some of the preceding lines were skipped due to log size limit`